lab-app/src/App.js

```diff
@@ -0,0 +1,49 @@
+import './App.css';
+import ironhackLogo from './images/ironhack-logo-xs.png'
+import menuTop from './images/menu-top-xs.png'
+import icon1 from './images/icon1.png'
+import icon2 from './images/icon2.png'
+import icon3 from './images/icon3.png'
+import icon4 from './images/icon4.png'
+
+
+function App() {
+  return (
+    <div className="App">
+      <section className='blueSection'>
+        <nav className='navBar'>
+          <img className='ironhackLogo' src={ironhackLogo} alt='ironhack-logo' />
+          <img className='menuTop' src={menuTop} alt='menu-top'/>
+        </nav>
+        <div className='hero'>
+          <h1>Say hello to ReactJS</h1>
+          <p>You will learn how to use the most popular frontend library, and to become a super ninja developer</p>
+          <button>Awesome!</button>
+        </div>
+      </section>
+      <section className='description'>
+        <div>
+          <img src={icon1} alt="" />
```

Comment:
Try to use the alts and avoid the empty ones as that would increase the usability of your application either when the image doesn't load or if you have a user that uses screen readers.

```suggestion
          <img src={icon1} alt="Declarative Icon" />
```